src/main/java/com/prgrms/needit/domain/notification/entity/Notification.java

```diff
@@ -67,11 +67,11 @@ public Notification(
 		this.notifiedContentType = notifiedContentType;
 		this.notifiedContentId = notifiedContentId;
```

Comment:
```suggestion
		this.contentType = contentType;
		this.contentId = contentId;
```
user에서도 notified 부분을 뺀 것 처럼 content도 똑같이 적용하면 좋을 것 같아요!! Notification 엔티티에서용

Reply:
아하 그러네요 반영하겠습니당!